src/frontend/web_application/src/components/Button/index.jsx

```diff
@@ -35,8 +37,9 @@ const Button = ({ children, className, icon, display, color, shape, responsive,
         'm-button--secondary': color === 'secondary',
         'm-button--success': color === 'success',
 
-        'm-button--expanded': display === 'expanded',
```

Comment:
It makes sense to me to keep expanded in display modes and duplicates css directives (it's not coherent to have `expanded display="inline"` for example):

```
&--inline-block {
  @include o-clickable--button;
}

&--expanded {
  @include o-clickable--button;
  @include o-clickable--expanded;
  width: 100%;  // useful ?
}
```